### DISCUSSION TITLE
post/ping-lun-qu-yu/

### DISCUSSION THREAD

**Discussion by giscus** [XMLC2024/xmlc2024.github.io#10] [Announcements]
# post/ping-lun-qu-yu/

https://xmlc2024.github.io/post/ping-lun-qu-yu/

**Comment by XMLC2024** [comment]
# 你好!
---
这里是对giscus的测试!

hello

**Reply by XMLC2024** [reply]
# 当前GISCUS的主题仍存在一些问题，麻烦你先《凑付着看》
## 如果你实在受不了，那就受不了吧?
